setup.cfg

```diff
@@ -20,19 +20,14 @@ python_requires = >=3.7
 install_requires =
     aiohttp>=3.6,<4
     protobuf>=3.0,<4
+    aiofiles>=0.6.0<1
```

Comment:
missing `,`

Reply:
the is equivalent to an `and` operator, so its not needed here as the full version is explicitly written.

Reply:
The other lines use a `,`, so it should do the same. Also it might be better to limit it to `0.7`, because 0.x semver may introduce breaking changes in minor version bumps